pkg/operator/staticpod/startupmonitor/cmd.go

```diff
@@ -54,7 +58,7 @@ type Options struct {
 	Check ReadinessChecker
 }
 
-func NewCommand(check ReadinessChecker) *cobra.Command {
+func NewCommand(check ReadinessChecker, newOperatorClient func(config *rest.Config) v1helpers.StaticPodOperatorClient) *cobra.Command {
```

Comment:
can `newOperatorClient` return an error ?

Reply:
no

Reply:
no? How are you planning to create the client? 

I think this is how it is created in `cas-o`

```
operatorClient, dynamicInformers, err := genericoperatorclient.NewStaticPodOperatorClient(controllerContext.KubeConfig, operatorv1.GroupVersion.WithResource("kubeapiservers"))
	if err != nil {
		return err
	}
```

where `NewStaticPodOperatorClient` actually creates:

```
dynamicClient, err := dynamic.NewForConfig(config)
	if err != nil {
		return nil, nil, err
	}
	client := dynamicClient.Resource(gvr)
```

Reply:
which is fine, no? `NewStaticPodOperatorClient` is what we have to call.

Reply:
fixed